tensorflow/tools/ci_build/builds/builds_common.sh

```diff
@@ -235,7 +235,7 @@ android_sdk_repository(
 android_ndk_repository(
     name="androidndk",
     path="${ANDROID_NDK_HOME}",
-    api_level=21)
+    api_level=25)
```

Comment:
ditto

Reply:
I have reverted the changes back. Thanks.